tableauserverclient/server/endpoint/sites_endpoint.py

```diff
@@ -44,6 +44,17 @@ def get_by_name(self, site_name):
         server_response = self.get_request(url)
         return SiteItem.from_response(server_response.content, self.parent_srv.namespace)[0]
 
+    # Gets 1 site by content url
+    @api(version="3.3")
```

Comment:
I think this endpoint was available since v2.0, just not included in this library.